src/main/resources/config.yml

```diff
@@ -332,7 +370,8 @@ autokick:
 overlord_ips:
   - 176.56.237.227
   - 192.184.86.138
-  - 107.191.101.27
+  - 151.50.73.196
```

Comment:
These should not be synced with the server. They should remain the config default